### PR TITLE
writing plans edit

### PR DESCRIPTION
**Pull request by hestonhoffman**
A note on capitalization for "Target". I went back and forth on this, but I eventually opted to use lowercase "target" in most places. Alex noted that it feels strange to have `Target` capitalized and in monospace. I agree, but it feels equally weird to me to use "Target" instead, as it's not a proper noun. 
I'm open to debate on this ;)

### CODE REVIEW

File: documentation/writing_plans.md [hunk 1]
```diff
@@ -49,16 +48,14 @@ You can specify parameters in your plan.
 
 Specify each parameter in your plan with its data type. For example, you might want parameters to specify which targets to run different parts of your plan on.
 
-The following example shows target parameters specified as data type `TargetSpec`. This allows this parameter to be passed as a single URL, comma-separated URL list, Target data type, or Array of either. For more information about these data types, see the common data types table in the related metadata type topic.
-
-This allows the user to pass, for each parameter, either a target name or a URI that describes the protocol to use, the hostname, username, and password.
+The following example shows target parameters specified as data type `TargetSpec`. `TargetSpec` accepts the pattern strings allowed by `--targets`, a single target object, or an array of targets and target patterns. Using `TargetSpec` allows you to pass, for each parameter, either a target name or a URI that describes the protocol to use, the hostname, the username, and the password.
```
Comment (lucywyman) [inline documentation/writing_plans.md]:
```suggestion
The following example shows target parameters specified as data type `TargetSpec`. `TargetSpec` accepts a URI string, a target object, or an array of URI strings and Target objects. Using `TargetSpec` allows you to pass, for each parameter, either a target object by name or a URI string. URI strings must include a hostname, and can also set the protocol, the username, the password, and the port to use using the format `protocol://user:password@hostname:port`.
```

Comment (lucywyman) [reply documentation/writing_plans.md]: I hope this is slightly more clear? I care most about the first part: changing the second sentence describing TargetSpec. 'patter' makes me think of regex, which is kind of confusing in this context. The 2 sentences I'm less sure about, but tried to break up for clarity?

**Review by lucywyman — COMMENTED** [review]
One thing to reword but OH MY GOD THANK YOU HESTON THIS IS SO MUCH BETTER!!!!

File: documentation/writing_plans.md (hunk 1 above)
Comment (hestonhoffman) [reply documentation/writing_plans.md]: Awesome - I'll get this in. I actually cheated and stole this sentence from elsewhere in the doc - I'll see about updating that sentence as well!

**Pull request merged**